imports/api/Players.js

```diff
@@ -3,4 +3,4 @@ import { Mongo } from 'meteor/mongo';
 //Until some actually halfway programming happens players 
 //must be created and edited through the actual database
 // >>This might not be a bad thing
-export const Players = new Mongo.Collection('players');
\ No newline at end of file
+export default new Mongo.Collection('players');
```

Comment:
How does this work?  Does importing Players from imports/api/Players.js just default to the name of the file?